### ISSUE TITLE
Druid Flight Form not covered

### ISSUE THREAD

**Issue by octomobiki** [Rottenbeer/ItemRack#179]
Can this be added to take advantage of mount gear swapping? 60% speed is slow enough as it is :~)

**Comment by Rottenbeer** [comment]
Yeah this works, just tell me the exact buff name or (better) druid stance index.

**Comment by octomobiki** [comment]
Not sure how I can tell, but if I'm following the other stances right I think it should just be 5, as it's fifth on the bar.

is there something I can run for you?

**Comment by Rottenbeer** [comment]
What's the exact buff name for the flight form?

**Comment by octomobiki** [comment]
![image](https://user-images.githubusercontent.com/24898483/122640267-a54b1880-d0cc-11eb-9066-0e99d2c6af4b.png)
![image](https://user-images.githubusercontent.com/24898483/122640270-aaa86300-d0cc-11eb-9861-8b41a8130b26.png)

Looks like "Flight Form"

**Comment by Rottenbeer** [comment]
[ItemRack-3.68-1-ga1c0e1d-bcc.zip](https://github.com/Rottenbeer/ItemRack/files/6680551/ItemRack-3.68-1-ga1c0e1d-bcc.zip)
Check this

**Comment by octomobiki** [comment]
I can see the options in the itemrack menu, however using "Flight Form" does not automatically switch

**Comment by octomobiki** [comment]
Is there anything else you'd like me to try?

**Comment by chad-hine** [comment]
I tried "stance = 5" too.  That didn't work either.
